### PR TITLE
Additional Features

### PR DESCRIPTION
**Pull request by Shubha-accenture**
Added the below features:
1. DPMS for Clusters and Serverless
2. Creating Runtime templates for serverless sessions
3. Added an item in the settings menu - Cloud Dataproc Settings

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -31,7 +31,7 @@ __pycache__/
 .Python
 build/
 develop-eggs/
-dist/
+# dist/
```
Comment (ptwng) [inline .gitignore:34]: Please undo, we aren't including binaries in the repo.

Comment (ojarjur) [reply .gitignore:34]: In addition to what @ptwng said, please also remove the `dist` directory from the repo and then rebase the commit to remove all commits where it was present.

File: src/dpms/tableInfo.tsx [hunk 2]
```diff
@@ -41,10 +41,9 @@ const TableInfo = ({
 }: IDatabaseProps): React.JSX.Element => {
   const table = {
     'Table name': title,
-    Description: tableDescription[title],
-    'By column name': '',
-    'By Database': database,
-    'By Dataproc Metastore Instance': dataprocMetastoreServices
+    'Description': tableDescription[title],
+    'Database': database,
+    'Dataproc Metastore Instance': dataprocMetastoreServices
```
Comment (jinnthehuman) [inline src/dpms/tableInfo.tsx:46]: Same thing here with the user friendly key names.

File: src/runtime/createRunTime.tsx [hunk 3]
```diff
@@ -0,0 +1,1176 @@
...
+  const handleSubNetworkChange = (event: any, data: any) => {
+    setSubNetworkSelected(data.value);
+  };
+  const handleCancelButton = () => {
+    setOpenCreateTemplate(false);
+    // const content = new AuthLogin();
+    // const widget = new MainAreaWidget<AuthLogin>({ content });
+    // widget.title.label = 'Config Setup';
+    // //widget.title.icon = iconCluster;
+    // app.shell.add(widget, 'main');
```
Comment (jinnthehuman) [inline src/runtime/createRunTime.tsx:685]: Delete

File: src/runtime/createRunTime.tsx [hunk 4]
```diff
@@ -0,0 +1,1176 @@
...
+              }
+            }
+          })
+        },
+        environmentConfig: {
+          executionConfig: {
+            ...(networkTagSelected.length > 0 && {
+              networkTags: networkTagSelected
+            }),
+            // ...(networkSelected && {networkUri:networkSelected}),
```
Comment (jinnthehuman) [inline src/runtime/createRunTime.tsx:818]: delete

**Review by jinnthehuman — CHANGES_REQUESTED** [review]

**Review by ptwng — APPROVED** [review]

**Review by jinnthehuman — APPROVED** [review]

**Pull request merged**